<ui version="4.0" >
 <class>View</class>
 <widget class="QMainWindow" name="View" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Charm</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QToolButton" name="goButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="stopButton" >
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="tasksCombo" >
        <item>
         <property name="text" >
          <string>All Tasks</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>Subscribed Tasks Only</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="buttonClearFilter" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filterLineEdit" />
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTreeView" name="treeView" >
      <property name="alternatingRowColors" >
       <bool>true</bool>
      </property>
      <property name="rootIsDecorated" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <tabstops>
  <tabstop>goButton</tabstop>
  <tabstop>stopButton</tabstop>
  <tabstop>tasksCombo</tabstop>
  <tabstop>filterLineEdit</tabstop>
  <tabstop>buttonClearFilter</tabstop>
  <tabstop>treeView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonClearFilter</sender>
   <signal>clicked()</signal>
   <receiver>filterLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>396</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel" >
     <x>429</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
